pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -92,37 +92,47 @@ For cloud installations, scaling out can be limited by resource quotas.
 In case you need to run Spark jobs on Kubernetes using Spark Operator, it needs to be installed separately.
 Use the following instructions to install Spark Operator from Kommander Catalog [DKP 2.x][install-spark-dkp2]
 
-## Install Kaptain
+## Add Kaptain to your DKP Catalog Applications via CLI 
 
-- Install the [kubectl-kudo CLI plugin][kudo_cli]
+If you installed DKP with Kaptain as a Workspace application in the Kommander installation file, you do not need to create a Git Repository for Kaptain. 
 
-- After the Konvoy cluster has been deployed (including Istio and KNative), install KUDO:
+If you added Kaptain after installing DKP, you must make it available by creating a Git Repository. Use the CLI to create the GitRepository resource and add a new repository. 
 
-  ```bash
-  kubectl kudo init --wait
-  ```
+### Create a Git repository for Kaptain
 
-- Download [kubeflow-1.4.0_1.3.0.tgz][download] tarball.
-<p class="message--note"><strong>NOTE: </strong>Starting with Kaptain 1.2.0, automatic profile creation on initial login is now disabled by default. See <a href="../../user-management">User Management</a> for more details.</p>
+1.  Refer to [air-gapped install instructions][airgapped_install], if you are running in air-gapped environment.
 
-- Set required configuration based on the target platform:
-  - When installing on Konvoy 1.x, add the following configuration to `parameters.yaml` file:
-  ```bash
-  cat >> parameters.yaml << END
-  dkpPlatformVersion: 1
-  installMinioOperator: true
-  END
-  ```
-- When installing on DKP 2.x, add the following configuration to `parameters.yaml` file:
+1.  Adapt the URL of your Git repository:
 
-  ```bash
-  # set the OIDC Provider CA bundle
-  OIDC_PROVIDER_CA_BUNDLE=$(kubectl get secret kommander-traefik-certificate -n kommander -o jsonpath="{.data.ca\.crt}")
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: kaptain-catalog-applications
+      labels: 
+        kommander.d2iq.io/gitrepository-type: catalog
+    spec:
+      interval: 1m0s
+      ref: 
+        branch: dev
+      timeout: 20s
+      url: https://github.com/mesosphere/kaptain-catalog-applications
+    EOF
+    ```
 
-  cat >> parameters.yaml << END
-  oidcProviderBase64CaBundle: ${OIDC_PROVIDER_CA_BUNDLE}
-  END
-  ```
+1.  Ensure the status of the `GitRepository` signals a ready state:
+
+    ```bash
+    kubectl get gitrepository example-repo -n ${PROJECT_NAMESPACE}
+    ```
+
+    The repository commit displays the ready state:
+
+    ```sh
+    NAME         URL                                                        READY   STATUS                                                              AGE
+    example-repo https://github.com/example-org/example-repo                True    Fetched revision: master/6c54bd1722604bd03d25dcac7a31c44ff4e03c6a   11m
+    ```
```

Comment:
I _think_ kaptain is installed at this point. commands below are more like deploying it. cc @lukeogg

Reply:
ok yeah... adding, installing, deploying.. we meant the same 🙈